extensions/resource-deployment/src/ui/notebookWizard/notebookWizardPage.ts

```diff
@@ -100,6 +100,8 @@ export class NotebookWizardPage extends ResourceTypePage {
 			await setModelValues(this._model.inputComponents, this.wizard.model);
 		}
 
+		this.wizard.wizardObject.generateScriptButton.enabled = this.pageObject.valid;
```

Comment:
I'd suggest moving this up to the `if (pageInfo.isLastPage) {` block - in the else for that we hide this button anyways and it'd be nice to keep all the button logic in one place.

Reply:
@Charles-Gagnon - as I mentioned earlier, hidden or not is a function of being on the last page or not while enablement is a function of page validity irrespective of the page we are on. So, the current code captures these semantics well. @aasimkhan30 - Kindly add a comment in the code that clearly spell out the above semantics so we maintain it going forward.